RecoTracker/DeDx/test/testProbQXY.sh

```diff
@@ -18,7 +18,7 @@ rm MiniAOD.root
 
 # test MINIAOD signal MC
 echo -e "TESTING  Getting probQ and probXY from signal MC MINIAOD ...\n\n"
-cmsDriver.py ReRECOonMC --conditions 106X_upgrade2018_realistic_v11_L1v1 --datatier MINIAOD --era Run2_2018,pf_badHcalMitigation --eventcontent MINIAOD --number 10 --python_filename 4HLT2MiniAODonMC.py --scenario pp --step RAW2DIGI,L1Reco,RECO,EI,PAT --runUnscheduled --mc  --filein  /store/user/tvami/HSCP/HSCPgluino_M_1800/crab_PrivateHSCP_2018_Gluino_Mass1800_HLT_NoPU_v1/210612_042006/0000/HSCP_Gluino_Mass1800_HLT_1.root  --fileout file:MiniAODonMC.root
+cmsDriver.py ReRECOonMC --conditions auto:phase1_2018_realistic --datatier MINIAOD --era Run2_2018 --eventcontent MINIAOD --number 10 --python_filename 4HLT2MiniAODonMC.py --scenario pp --step RAW2DIGI,L1Reco,RECO,PAT --mc  --filein  /store/user/tvami/HSCP/HSCPgluino_M_1800/crab_PrivateHSCP_2018_Gluino_Mass1800_HLT_NoPU_v1/210612_042006/0000/HSCP_Gluino_Mass1800_HLT_1.root  --fileout file:MiniAODonMC.root
```

Comment:
I don't think cmsbuild machines can read `/store/user/` areas.
Also I don't think it's good practice to use personal accounts areas in central tests.

Reply:
I asked @smuzaffar to move it to the central ibeos cache yesterday, and he said it's been copied. How should I change the test it reads that version instead of mine?

Reply:
What's the address of the new location?

Reply:
The file 
`/eos/cms/store/user/cmsbuild/store/user/tvami/HSCP/HSCPgluino_M_1800/crab_PrivateHSCP_2018_Gluino_Mass1800_HLT_Small_v1/211208_211538/0000/HSCP_Gluino_Mass1800_HLT_1.root `
indeed exists

Reply:
Ok then. Still looks awkward to have `tvami` in it.

Reply:
And I also saw another user under `/eos/cms/store/user/cmsbuild/store/user/`
Let me also add that I'd be the happiest if central MC simulations for this signal would be stored in RAW format (and so my private samples would not be needed), this of course is not done due to the usual space requirement.